Need to add explanation of script behavior for AppGW backend pool  - Update public-ip-upgrade-vm.md

maotsuk opened the pull request:
CSS has received multiple cases where customers report that the script does not account for NICs assigned to the AppGW backend pool. If a NIC with a Basic Public IP is upgraded using this script, it will be removed from the AppGW backend pool. Customers must manually reassign the NIC after the script runs — but many are unaware of this until it's too late. Therefore, I strongly recommend adding this Q&A to the MS Learn page to help prevent confusion and reduce support cases.

maotsuk committed:
Update public-ip-upgrade-vm.md

CSS has received multiple cases where customers report that the script does not account for NICs assigned to the AppGW backend pool.
If a NIC with a Basic Public IP is upgraded using this script, it will be removed from the AppGW backend pool.
Customers must manually reassign the NIC after the script runs — but many are unaware of this until it's too late.
Therefore, I strongly recommend adding this Q&A to the MS Learn page to help prevent confusion and reduce support cases.

mbender-ms commented on articles/virtual-network/ip-services/public-ip-upgrade-vm.md:
```diff
@@ -108,6 +108,11 @@ There is no way to evaluate upgrading a Public IP without completing the action.
 
 Yes, the process of upgrading a Zonal Basic SKU Public IP to a Zonal Standard SKU Public IP is identical and works in the script.
 
+### If I specify a NIC associated with a public IP targeted for migration in the AppGW backend pool, will this script remove it from the pool?
```
```suggestion
### If I specify a NIC associated with a public IP targeted for migration in the Application Gateway backend pool, will this script remove it from the pool?
```

mbender-ms commented on articles/virtual-network/ip-services/public-ip-upgrade-vm.md:
```diff
@@ -108,6 +108,11 @@ There is no way to evaluate upgrading a Public IP without completing the action.
 
+### If I specify a NIC associated with a public IP targeted for migration in the AppGW backend pool, will this script remove it from the pool?
+
+Yes, it will be removed. After running the script, you will need to manually reassign the NIC to the AppGW backend pool.
```
```suggestion
Yes, it will be removed. After running the script, you will need to manually reassign the NIC to the Application Gateway backend pool.
```

mbender-ms reviewed (changes requested):
@maotsuk  Please accept the suggestions for spelling out Application Gateway.

maotsuk committed:
Update articles/virtual-network/ip-services/public-ip-upgrade-vm.md

Thank you for your confirmation!
" AppGW = Application Gateway " so, I accept this suggestion!

Co-authored-by: Michael Bender <102542398+mbender-ms@users.noreply.github.com>

v-dirichards committed:
Update articles/virtual-network/ip-services/public-ip-upgrade-vm.md

Co-authored-by: Michael Bender <102542398+mbender-ms@users.noreply.github.com>
